components/multimedia/youtube-dl/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Nona Hansel
+#
+
+BUILD_BITS=		NO_ARCH
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		youtube-dl
+COMPONENT_VERSION=	2020.1.24
+COMPONENT_TMP_VERSION=	2020.01.24
+COMPONENT_SUMMARY=	'Command-line program to download videos from YouTube.com and other video sites'
+COMPONENT_PROJECT_URL=	https://youtube-dl.org/
+COMPONENT_FMRI=		multimedia/video/youtube-dl	
+COMPONENT_CLASSIFICATION= Applications/Sound and Video
+COMPONENT_SRC=          $(COMPONENT_NAME)
+COMPONENT_ARCHIVE=      $(COMPONENT_NAME)-$(COMPONENT_TMP_VERSION).tar.gz
+COMPONENT_ARCHIVE_URL=	$(COMPONENT_PROJECT_URL)/downloads/latest/$(COMPONENT_ARCHIVE)	
+COMPONENT_ARCHIVE_HASH= \
+	sha256:0b1dcc96d204984df1ee364e81cddd5645421f6065d2815472258ccdad5c36ff
+COMPONENT_LICENSE=	Unlicense
+COMPONENT_LICENSE_FILE=	LICENSE
+
+PYTHON_VERSIONS=	3.5
+
+include $(WS_MAKE_RULES)/common.mk
```

Comment:
Add `TEST_TARGET=$(NO_TESTS)` above `common.mk` include if there is no test suite.

Reply:
Did it.